src/scenes/level.js

```diff
@@ -90,9 +102,12 @@ export class LevelScene extends Phaser.Scene {
         // Exit button
         this.load.image("exit-button", "src/assets/ui/buttons/ExitButton.png");
 
-        // Click object (target)
-        this.clickObjectMetaData.forEach(target => {
-            this.load.image(target.name, target.path);
+        // Targets
+        this.targetMetaData.forEach(target => {
+            let test = new target(this);
+            test.images.forEach(image => {
+                this.load.image(image.name, image.path);
+            });
```

Comment:
What is the extra test variable doing? Are the objects stored in targetMetaData not already target objects?

Reply:
Rip forgot to change that. Interestingly enough they are uninitialized objects, until they are called with the "new" keyword the class is stored as what seems like text and its member attributes cannot be accessed until you initialize it.

I would like to add the objects into `targets[]`  here but in their constructor they call the `this.add.image()` function. I don't think you can do this during in the `preload` function so I just load the images here and then later on create the actual targets. 

I can think of 2 ways to address this:
1) just renamed test to temp and not worry about it
2) Move the `this.add.image()` into a separate create function under the Target class. Add the targets into the array here and then again in the create function of the Level class call each target's create function

Reply:
I like the idea of the targets[] array, but if this is the only place you'd use it then it's fine to leave it as-is

Reply:
I tried adding it here then creating it later but it threw some other errors because of some other dependencies so I'm just gonna rename it and leave it as is I think.